pkg/parser/translate_test.go

```diff
@@ -725,7 +726,7 @@ func TestFromTCPIngressV1beta1(t *testing.T) {
 				Name:      "foo",
 				Namespace: "default",
 			},
-			Spec: configurationv1beta1.IngressSpec{
+			Spec: configurationv1beta1.TCPIngressSpec{
```

Comment:
Non-issue, but noting down since I was confused originally: the only spec that we defined originally in our own configurationv1beta1 was for TCPIngress, but we didn't name it accordingly. These types were _not_ originally the stock K8S IngressSpec.

Reply:
:+1: